packages/styles/tabs.css

```diff
@@ -0,0 +1,89 @@
+:root {
+  --tabs-border-color: #b3bfc6;
+  --tab-shadow-color: var(--accent-primary);
+  --tab-inactive-background-color: var(--gray-20);
+  --tab-inactive-text-color: var(--gray-60);
+  --tab-active-background-color: var(--white);
+  --tabs-active-text-color: var(--gray-90);
+}
+
+.cauldron--theme-dark {
+  --tabs-border-color: #5d676f;
+  --tab-shadow-color: var(--accent-info);
+  --tab-inactive-background-color: var(--accent-medium);
+  --tab-inactive-text-color: var(--accent-light);
+  --tab-active-background-color: var(--accent-dark);
+  --tabs-active-text-color: var(--white);
+}
+
+.Tabs {
+  width: fit-content;
+}
+
+.Tablist {
+  display: flex;
+  flex-direction: row;
+  border-left: 1px solid var(--tabs-border-color);
+}
+
+.Tab {
+  display: block;
+  text-align: center;
+  height: 2.875rem;
+  width: 9.375rem;
```

Comment:
NOT NECESSARILY BLOCKING:

I think that we should change this. Anything beyond the width gets truncated, and I do think if we have space to display the text in the tabs we should display it instead of just ellipsizing the text. This leads to a loss of content that there is no alternative way to view.

Reply:
Yeah, we need to discuss which is the better approach.  I think in most cases we should be able to extend the width of the Tab and show the entire title, but we do want to prevent the Tab title from getting too long. @bobbyomari @awpearlm